data/json/items/armor/coats.json

```diff
@@ -1424,7 +1473,7 @@
     "material": [ "cotton" ],
     "symbol": "[",
     "looks_like": "house_coat",
-    "color": "black",
+    "color": "dark_gray",
```

Comment:
You'll want to change the description from "jet black".

Reply:
Oh, I changed the color because it seems like "black" makes the item icon invisible (because the panel color is black). But I still visualize it as "jet black". Some of my new items were `"color": "black"` too, but it didn't work.

Reply:
Makes sense.